giftcard.adoc

```diff
@@ -0,0 +1,46 @@
+## Giftcard API
+
+### Service Description
+The Giftcard API provides extra information about purchases made through iZettle with a giftcard.
+
+### URL
+https://giftcard.izettle.com
+
+### Scopes
+The Giftcard API implements the following scopes:
+
+- `READ:PURCHASE`
```

Comment:
Maybe `READ:FINANCE` (used when fetching account statements from finance-service) is more appropriate if we are to reuse any of the existing scopes?

Reply:
Yeah it feels more appropriate I think if we want to reuse existing scope. 

How much work is it to introduce a new scope since probably the right way is to introduce a `READ:GIFTCARD` but it might be overkill?

Reply:
Unfortunately I don't know of the tech changes and potential merchant actions that are needed to add a new scope, but I think @psvanstrom should be able to share some insight into it?!

Reply:
After some discussions with @psvanstrom and @thiyagugk the following can be said.

Adding a new scope is from our point a small thing but it can be a huge thing for merchants and integrators.
Current api is closely tied to purchase so keep that for now. If we in the future feel the need of a giftcard scope we can add it and initially have both scopes but phase out the purchase scope. 

In short we're keeping `READ:PURCHASE`

Reply:
Sounds good to me, nice explanation of reasoning behind decision! :)